### ISSUE TITLE
punctuation missing

### ISSUE THREAD

**Issue by thanhlong1997** [VinAIResearch/XPhoneBERT#6]
Hi, I am trying Xphonebert for Vietnamese TTS system and I find that Xphonebert is simply skip punctuation character when convert input sequence to phoneme sequence by using Text2PhonemeSequence library
For example:
```python
import torch
from transformers import AutoModel, AutoTokenizer
from text2phonemesequence import Text2PhonemeSequence

# Load XPhoneBERT model and its tokenizer
xphonebert = AutoModel.from_pretrained("vinai/xphonebert-base")
tokenizer = AutoTokenizer.from_pretrained("vinai/xphonebert-base")

# Load Text2PhonemeSequence
text2phone_model = Text2PhonemeSequence(language="vie-n", is_cuda=False)

# Input sequence that is already word-segmented (and text-normalized if applicable)
sentence1 = "dù sao tiền cũng đã trả rồi, chờ xem phản ứng từ thị trường thế nào đã rồi nói tiếp."
sentence2 = "dù sao tiền cũng đã trả rồi. chờ xem phản ứng từ thị trường thế nào đã rồi nói tiếp."
input_phonemes1 = text2phone_model.infer_sentence(sentence1)
input_phonemes2 = text2phone_model.infer_sentence(sentence2)
``` 
The phoneme sequence of 2 input is the same:
```python
z u ˧˨ ▁ s a w ˧˧ ▁ t i ə n ˧˨ ▁ k u ŋ͡m ˧ˀ˥ ▁ d a ˧ˀ˥ ▁ c a ˧˩˨ ▁ z o j ˧˨ ▁ c ɤ ˧˨ ▁ s ɛ m ˧˧ ▁ f a n ˧˩˨ ▁ ɯ ŋ ˨˦ ▁ t ɯ ˧˨ ▁ tʰ i ˨ˀ˩ ʔ ▁ c ɯ ə ŋ ˧˨ ▁ tʰ e ˨˦ ▁ n a w ˧˨ ▁ d a ˧ˀ˥ ▁ z o j ˧˨ ▁ n ɔ j ˨˦ ▁ t i ə p ˦˥
``` 
This will raise an misunderstanding for model to learn break between sentence parts. 
Pls check it out !!! 
Thank you !!!

**Comment by datquocnguyen** [comment]
As stated in the Readme file, you have to perform Vietnamese word segmentation. 
The input should be: `dù_sao tiền cũng đã trả rồi , chờ xem phản_ứng từ thị_trường thế_nào đã rồi nói tiếp .`

**Comment by manhcuong17072002** [comment]
@thanhlong1997 Have you been training it? What performance have you achieved with it?

**Comment by manhcuong02** [comment]
@thanhlong1997 Have you been training it? What performance have you achieved with it? Is the result good?

**Comment by thanhlong1997** [comment]
> @thanhlong1997 Have you been training it? What performance have you achieved with it? Is the result good?

Not good as I expected. I think the most important thing if you want to train a mutilingual TTS is data, much more data in difference language is good enough. if you use this XphoneBert even you don't have the data in the regional, It will be failed. And if you have enough data from various language, you don't need XphoneBert, only character is enough

**Comment by manhcuong02** [comment]
@thanhlong1997 I am training it on a Vietnamese dataset with around 2 hours of audio (as the author mentioned, 0.9 hours should be good enough). However, when I freeze the XphonemeBert model, I get poor results. It only stutters and doesn’t perform as expected. When XphonemeBert is trained together with the TTS model, it hardly produces anything, just noise. Have you encountered this issue before?

**Comment by thanhlong1997** [comment]
> @thanhlong1997 I am training it on a Vietnamese dataset with around 2 hours of audio (as the author mentioned, 0.9 hours should be good enough). However, when I freeze the XphonemeBert model, I get poor results. It only stutters and doesn’t perform as expected. When XphonemeBert is trained together with the TTS model, it hardly produces anything, just noise. Have you encountered this issue before?

I don't think you need to freeze or train XphoneBert. simply convert your text label into list of phoneme using pre-train XphoneBert. Do not use Xphonebert as a layer in your model, consider it as pre-processing step. just use to pre-process text only.
